lib/jekyll-feed/generator.rb

```diff
@@ -48,6 +49,22 @@ def feed_path(collection: "posts", category: nil)
       collections.dig(collection, "path") || "#{prefix}.xml"
     end
 
+    # Determines the title of a given feed
+    #
+    # collection - the name of a collection, e.g., "posts"
+    # category - a category within that collection, e.g., "news"
+    #
+    # Will return the site title or name
+    # ...followed by collection title or capitalized name
+    # ...followed by capitalized category name
+    def feed_title(collection: "posts", category: nil)
+      words = []
+      words << (@site.config["title"] || @site.config["name"])
+      words << collections.dig(collection, "title") || collection.capitalize unless collection == "posts" # rubocop:disable Metrics/LineLength
```

Comment:
```suggestion
      unless collection == "posts"
        words << collections.dig(collection, "title") || collection.capitalize
      end
```

Reply:
Indeed, this is more readable. But we need parenthesis like above because of operator precedence, i.e: ` words << (collections.dig(collection, "title") || collection.capitalize)`

Reply:
I see. I've updated my suggestion. You can either commit that or push a local commit.
Entirely your call.